content/en/agent/guide/how_remote_config_works.md

```diff
@@ -31,7 +31,7 @@ Remote configuration is not available on the US1-FED Datadog site.
 <div class="alert alert-info">Remote Configuration is in beta.</a></div>
 
 ## Overview
-Remote Configuration is a Datadog capability that allows you to remotely configure the behavior of Datadog resources (for example, Agents and tracing libraries) deployed in your infrastructure, for select product features. Using Remote Configuration makes it easier to apply Agent and tracing library configurations in your environment on demand, decreasing management costs, reducing friction between teams, and accelerating resolution times. 
+Remote Configuration is a Datadog capability that allows you to remotely configure the behavior of Datadog resources (for example, Agents, tracing libraries, and Observability Pipelines Worker) deployed in your infrastructure, for select product features. Using Remote Configuration makes it easier to apply configurations to Datadog resources in your environment on demand, decreasing management costs, reducing friction between teams, and accelerating issue resolution times. 
```

Comment:
```suggestion
Remote Configuration is a Datadog capability that allows you to remotely configure the behavior of Datadog resources deployed in your infrastructure. You can configure select product features on your Datadog Agents, tracing libraries, and the Observability Pipelines Worker. Use Remote Configuration to apply configurations to Datadog resources in your environment on demand, decreasing management costs, reducing friction between teams, and accelerating issue resolution times.
```

Reply:
Thx Heston for the review. On this one, the select features are not configured on Datadog Agents, tracing libraries, and OPW but rather the products themselves such as APM, ASM, etc. Also it's important to clarify what resources Datadog resources are inline. How about this?

Remote Configuration is a Datadog capability that allows you to remotely configure the behavior of Datadog resources (for example, Agents, tracing libraries, and Observability Pipelines Worker) deployed in your infrastructure, for select product features. Use Remote Configuration to apply configurations to Datadog resources in your environment on demand, decreasing management costs, reducing friction between teams, and accelerating issue resolution times.